resources/js/vat-validation.js

```diff
@@ -0,0 +1,123 @@
+import { useThrottleFn } from '@vueuse/core'
+import { token } from 'Vendor/rapidez/core/resources/js/stores/useUser'
+import { mask } from 'Vendor/rapidez/core/resources/js/stores/useMask'
+
+const patterns = {
+    AT: /^U[A-Z\d]{8}$/,
+    BE: /^[0-2]\d{9}$/,
+    BG: /^\d{9,10}$/,
+    CY: /^\d{8}[A-Z]$/,
+    CZ: /^\d{8,10}$/,
+    DE: /^\d{9}$/,
+    DK: /^(\d{2} ?){3}\d{2}$/,
+    EE: /^\d{9}$/,
+    EL: /^\d{9}$/,
+    ES: /^([A-Z]\d{7}[A-Z]|\d{8}[A-Z]|[A-Z]\d{8})$/,
+    EU: /^\d{9}$/,
+    FI: /^\d{8}$/,
+    FR: /^[A-Z\d]{2}\d{9}$/,
+    GB: /^(\d{9}|\d{12}|(GD|HA)\d{3})$/,
+    HR: /^\d{11}$/,
+    HU: /^\d{8}$/,
+    IE: /^([A-Z\d]{8}|[A-Z\d]{9})$/,
+    IT: /^\d{11}$/,
+    LT: /^(\d{9}|\d{12})$/,
+    LU: /^\d{8}$/,
+    LV: /^\d{11}$/,
+    MT: /^\d{8}$/,
+    NL: /^\d{9}B\d{2}$/,
+    PL: /^\d{10}$/,
+    PT: /^\d{9}$/,
+    RO: /^\d{2,10}$/,
+    SE: /^\d{12}$/,
+    SI: /^\d{8}$/,
+    SK: /^\d{10}$/,
+    SM: /^\d{5}$/,
+}
+
+document.addEventListener('turbo:load', function () {
```

Comment:
Note that we have also got the `vue:loaded` event in case we want to make sure Vue is available on the request:
https://github.com/rapidez/core/blob/667c18a698f93cd253825530ed75ca98c29ff52b/resources/js/package.js#L160